### PR TITLE
Fixed Order model lose `created_at` property.

### PR DESCRIPTION
**Pull request by hiscaler**

### CODE REVIEW

**Comment by maximzasorin** [comment]
Hello, thanks for your commit. In your commit you have added a receipt field definition for the Fulfillment model. Where did you get this definition? The API documentation has a completely [different definition](https://help.shopify.com/en/api/reference/shipping-and-fulfillment/fulfillment#receipt-property-2019-10), maybe there is an error in the documentation, but how to check yours? How can I create Fulfillment with receipt field?

**Comment by hiscaler** [comment]
@maximzasorin I get from orders admin api. response data like follow:
```json
{
    "receipt": {
        "gift_cards": [
            {
                "id": 111,
                "line_item_id": 222,
                "masked_code": "•••• •••• •••• 333"
            }
        ]
    }
}
```

**Comment by maximzasorin** [comment]
@hiscaler 

I think we should clarify with the support or somewhere else this discrepancy. Maybe the value of this field does not have a constant format and depends on something, for example, as in the case of the receipt field for Transaction: https://help.shopify.com/en/api/reference/orders/transaction#receipt-property-2019-10

**Comment by hiscaler** [comment]
@maximzasorin Maybe, but i not add it, parse will be throw error.

File: src/Manager/Order/GiftCard.php [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    /**
+     * @return int
+     */
+    public function getId()
+    {
+        return $this->id;
+    }
+
+    /**
+     * @param int $id
```
Comment (jstoone) [inline src/Manager/Order/GiftCard.php:43]:
```suggestion
     * @param int $id
     *
     * @return GiftCard
```

File: src/Manager/Order/GiftCard.php [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+     */
+    public function setId($id)
+    {
+        $this->id = $id;
```
Comment (jstoone) [inline src/Manager/Order/GiftCard.php:47]:
```suggestion
        $this->id = $id;
        
        return $this;
```

File: src/Manager/Order/GiftCard.php [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    /**
+     * @return int
+     */
+    public function getLinnItemId()
+    {
+        return $this->linnItemId;
+    }
+
+    /**
+     * @param int $linnItemId
```
Comment (jstoone) [inline src/Manager/Order/GiftCard.php:59]:
```suggestion
     * @param int $linnItemId
     *
     * @return GiftCard
```

File: src/Manager/Order/GiftCard.php [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+     */
+    public function setLinnItemId($linnItemId)
+    {
+        $this->linnItemId = $linnItemId;
```
Comment (jstoone) [inline src/Manager/Order/GiftCard.php:63]:
```suggestion
        $this->linnItemId = $linnItemId;
        
        return $this;
```

File: src/Manager/Order/GiftCard.php [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+    /**
+     * @return string
+     */
+    public function getMaskedCode()
+    {
+        return $this->maskedCode;
+    }
+
+    /**
+     * @param string $maskedCode
```
Comment (jstoone) [inline src/Manager/Order/GiftCard.php:75]:
```suggestion
     * @param string $maskedCode
     *
     * @return GiftCard
```

File: src/Manager/Order/GiftCard.php [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+     */
+    public function setMaskedCode($maskedCode)
+    {
+        $this->maskedCode = $maskedCode;
```
Comment (jstoone) [inline src/Manager/Order/GiftCard.php:79]:
```suggestion
        $this->maskedCode = $maskedCode;
        
        return $this;
```

**Review by jstoone — CHANGES_REQUESTED** [review]
### General
These are great changs! :tada:

In #90 I added missing return statements to the setters of other models, which allows for a fluent method chaining. So I've suggested some changes to align your changes with mine. :relaxed: :v:

**Comment by slince** [comment]
2.x version is coming to an end; can you send pr to 3.x branch？

**Pull request closed without merging**